### DISCUSSION TITLE
How to set up triggers?

### DISCUSSION THREAD

**Discussion by AumCoin** [keepassxreboot/keepassxc#8299] [Q&A] [answered]
I came to KeepassXC because for some reason Keepass doesn't work on my Mac, and I figured I would at least try another version before ditching my Macbook and buying another laptop.  But I see that there's no triggers to help me autosync via WebDav.  For that matter, I can't figure out how to sync via WebDav.  Does this software have WebDav or triggers, like the original software it is supposedly based on?

**Comment by droidmonkey** [comment, marked as answer]
No we do not support webdav nor triggers. We also do not share any code with KeePass original, we fully support the KeePass KDBX standard though for opening and saving databases. 

Both of those functions you list are actually the responsibility of the operating system or even external tools. We have taken the stance of building a _password manager_ and not a all in one tool.

You can easily mount webdav folders in Finder and then open and save your database directly.
 https://support.apple.com/guide/mac-help/connect-disconnect-a-webdav-server-mac-mchlp1546/mac

Although we suggest using a modern cloud sync tool that supports version history (backups) and conflict management.
